test/MenuTest/Feelgood/feelGood_test.dart

```diff
@@ -92,13 +96,13 @@ void main() {
           .pumpWidget(getMenuForTests(mockUserInformation, mockAppInformation));
 
       await tester.tap(find.text('תפריט'));
-      await tester.pumpAndSettle(const Duration(seconds: 1));
+      /*  await tester.pumpAndSettle(const Duration(seconds: 1));
```

Comment:
why is this commented?

Reply:
Was commented by mistake, now it's fixed